### ISSUE TITLE
Startup message prevents non-interactive use in scripts

### ISSUE THREAD

**Issue by rossmeissl** [HubSpot/cos_uploader#12]
My deploy script (which uses this tool in this build pipeline) stopped working recently. When I run the `upload_to_cos` step manually, I get:

``` shell
 ~/code/glitz  ⑂ master    upload_to_cos --api-key=[elided] --action=upload --hub-id=[elided] -t preflight/
-------------------------
Due to authentication infrastructure changes the week of May 11, access tokens were reset. If you delete your .cos-sync-config.yaml file and rerun the uploader, you should be prompted to reauthenticate yourself.
-------------------------
Press ctrl-c to quit, or press any key to continue ...
```

Can this "startup message" be disabled?

**Comment by brilee** [comment]
Hey Andy, thanks for the feedback. I was considering just removing this line - https://github.com/HubSpot/cos_uploader/blob/688b31cef47cc0332cae8af9ee6ce453ad2aa37a/cos_uploader/upload_to_cos.py#L61 - that should fix your issue; and I don't think the "extra" interactivity adds much to the tool, in any case.

**Comment by rossmeissl** [comment]
@brilee please do!

**Comment by rossmeissl** [comment]
Fixed in 3f79cc4802d534f126d574449ef6dc165df9ac18
